lib/vm/Cargo.toml

```diff
@@ -21,7 +21,7 @@ more-asserts = "0.2"
 cfg-if = "1.0"
 backtrace = "0.3"
 serde = { version = "1.0", features = ["derive", "rc"] }
-rkyv = { version = "0.7.20", optional = true }
+rkyv = { version = "0.7.38", features = ["indexmap"] }
```

Comment:
It seems strange that the vm crate needs any serializable types. The vm crate is not used by the compiler, so how could the compiler include any of these types in the serialized artifact?